auth_otp/admin.py

```
@@ -1,3 +1,242 @@
+        'user',
+        'verified_at',
+        'access_token',
+        'refresh_token',
+        'device_id',
+        'device_name',
+        'session_key'
+    ]
+    
+    ordering = ['-verified_at']
+    
+    def get_phone_number(self, obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_phone_number`

(ANN201)